admin-dev/themes/default/sass/partials/_nav.sass

```
@@ -144,6 +144,18 @@
 	padding: 0 0 5.313rem 0
 	margin: 0
 
+	&.sidebar-closed
+		.link-levelone
+			.link > span
+				display: none;
```

```suggestion
				display: none
```
You are in `.sass` file. Please don't use semicolon.